WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -117,7 +118,12 @@ final class BlogDashboardViewController: UIViewController {
     }
 
     private func addQuickStartObserver() {
-        NotificationCenter.default.addObserver(self, selector: #selector(toggleQuickStart), name: .QuickStartTourElementChangedNotification, object: nil)    }
+        NotificationCenter.default.addObserver(self, selector: #selector(loadCardsFromCache), name: .QuickStartTourElementChangedNotification, object: nil)
+    }
+
+    private func addShowScheduledCardsObserver() {
```

Comment:
What do you think of calling this `addUpdateDashboardObserver()`? Since the `.updateDashboad` notification is posted for both drafts and scheduled posts
```suggestion
    private func addUpdateDashboardObserver() {
```

Reply:
Done in 411536c5463a17a6285c9d37649ac26b53c53e72